pkgs/development/python-modules/llama-cpp-python/default.nix

```diff
@@ -72,7 +62,10 @@ buildPythonPackage rec {
     # -mcpu, breaking linux build as follows:
     #
     # cc1: error: unknown value ‘native+nodotprod+noi8mm+nosve’ for ‘-mcpu’
-    [ "-DGGML_NATIVE=off" ]
+    [
+      "-DGGML_NATIVE=off"
+      "-DGGML_BUILD_NUMBER=1"
```

Comment:
If you `deepClone` then the version will actually be extracted from the (submodule) git revision; and then you don't need to override it with the variable here. These two are mutually exclusive.